packages/legacy-client/src/oauth-client/PublicClient/PublicClientFlowProvider.ts

```diff
@@ -75,6 +75,12 @@ export async function getTokenWithCodeVerifier(
   multipassContextPath?: string,
 ): Promise<OAuthToken> {
   const parsedUrl = new URLSearchParams(url.substring(url.indexOf("?")));
+
+  const error = parsedUrl.get("error");
+  if (error) {
+    throw new Error("Error received from server: " + error);
+  }
+
   const code = parsedUrl.get("code");
   if (!code) {
     throw new Error("Code parameter missing in URL: " + url);
```

Comment:
If we receive the callback with the error in the url query param then we would have failed here in the past so I included the actual error if it's present so in the default template the user will at least see the real reason for the code parameter being missing
![image](https://github.com/palantir/osdk-ts/assets/13809355/6ac39f1a-798f-49c4-b626-214dd9620e38)